tests/test_samples.py

```diff
@@ -54,6 +54,22 @@ def test_samples_plot(geom, to_funvals, plot_par):
     with pytest.raises(ValueError,match=r"Cannot pass is_par as a plotting argument"):
         s.plot(is_par=not to_funvals,plot_par=plot_par)
 
+@pytest.mark.parametrize("geom",[
+						(cuqi.geometry.Discrete(1)),
+						(cuqi.geometry.Continuous1D(1))
+						])
+@pytest.mark.parametrize("kwargs",[
+                        ({}),
+                        ({"marker":"o"}),
```

Comment:
Just a small suggestion, could use something other than the default option here, for example 
```suggestion
                        ({"marker":"*"}),
```